### ISSUE TITLE
generalize DarkSwitch to ThemeSwitch

### ISSUE THREAD

**Issue by McNerdius** [McNerdius/TailBlazor#25]
Currently, DarkSwitch is used with Tailwind's `class` [dark mode](https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually).  All it really does is apply a class to `<body>`, which would play well with CSS variables:

Each theme would have a css class name to apply to `<body>` and what is essentially a RenderFragment:  Currently SVGs and `light` / `dark` / `system`.  Generalizing this to Themes rather than just dark/light, the C# and JS would be largely the same, but the UI would differ - perhaps a dropdown if in a menu, or radio buttons if in a settings page.  The RenderFragments would be text instead of the current light/system/dark SVGs.

tl;dr: arbitrary UI; click on a RenderFragment and its associated class name is applied to `<body>`

**Comment by McNerdius** [comment]
On second thought, while it's a good idea in the context of useful Components, the amount of complexity it would add to this project is too much to demonstrate a simple aspect of Tailwind: ability to use `var(--some-variable)`.  Keep it simple.
